File: week-3/student.cpp
#include<iostream>
using namespace std;
class Student
{
public:
string fullName;
int rollNum;
double semPercentage;
string collegeName;
int collegeCode;
Student()
{
cout<<"Student class initialized!!\nIam a student\n";
}
~Student(){
  
}
int main()
{
Student sanjana;
cout<<"Enter your name : ";
getline(cin,sanjana.fullName);
cout<<"Enter your roll number : ";
cin>>sanjana.rollNum;
cout<<"Enter your sem percentage : ";
cin>>sanjana.semPerentage;
cout<<"Enter your college name : ";
cin>>sanjana.collegeName;
cout<<"Enter your college code : ";
cin>>sanjana.collegeCode;
cout<<"Name : "<<sanjana.fullName<<endl;
cout<<"RollNum : "<<sanjana.rollNum<<endl;
cout<<"SemPercentage : "<<sanjana.semPercentage<<endl;
cout<<"CollegeName : "<<sanjana.collegeName<<endl;
cout<<"CollegeCode : "<<sanjana.collegeCode<<endl;
return 0;
}

File: week-4 /student.java
double semPercentage;
    String collegeName;
    int collegeCode;
    Student65()
    {
      System.out.println("Student class is created successfully!!\nIam a student\n");
    }
    protected void finalize() throws Throwable
    {
      System.out.println("The class is finalized\n");
    }
}

public class Student
{
  public static void main(String[] args)
  {
    Student65 sanjana=new Student65();;
    Scanner input=new Scanner(System.in);
    System.out.print("Enter your name : ");
    sanjana.fullName=input.nextLine();
    System.out.print("Enter your rollNumber : ");
    sanjana.rollNum=input.nextInt();
    System.out.print("Enter your semPercentage : ");
    sanjana.semPercentage=input.nextDouble();
    input.nextLine();
    System.out.print("Enter your collegeName : ");
    sanjana.collegeName=input.nextLine();
    System.out.print("Enter your collegeCode : ");
    sanjana.collegeCode=input.nextInt();
    System.out.print("Name : "+sanjana.fullName+"\n");
    System.out.print("RollNumber : "+sanjana.rollNum+"\n");
    System.out.print("SemPercentage : "+sanjana.semPercentage+"\n");
    System.out.print("CollegeName : "+sanjana.collegeName+"\n");
    System.out.print("CollegeCode : "+sanjana.collegeCode+"\n");
    input.close();
  }
}
